scripts/bump_version.sh

```diff
@@ -96,7 +96,7 @@ then
   mv tmp library/CMakeLists.txt
 
   [ $VERBOSE ] && echo "Bumping SOVERSION for libmbedcrypto in library/Makefile"
-  sed -e "s/SOEXT_CRYPTO=so.[0-9]\{1,\}/SOEXT_CRYPTO=so.$SO_CRYPTO/g" < library/Makefile > tmp
+  sed -e "s/SOEXT_CRYPTO?=so.[0-9]\{1,\}/SOEXT_CRYPTO?=so.$SO_CRYPTO/g" < library/Makefile > tmp
```

Comment:
does `sed` need the `?` character to be escaped? IIRC there are different versions of the regex grammar for `sed`, and maybe different versions in different genealogies of `sed`, so I'm not sure what's required here

Reply:
No, `?` is not in the basic regular expression syntax, which is what `sed` uses by default.

Reply:
ah, okay, looks like `sed` typically uses Basic Regular Expressions, where `?` is a plain character, so this RE is fine